pkg/lib/web/image.go

```diff
@@ -510,3 +510,175 @@ func (a LocaleAwareImageDescriptor) GetSizeLimit() int {
 	}
 	return a.SizeLimit
 }
+
+var staticImageRegex = regexp.MustCompile(`^static/(.+)\.(png|jpe|jpeg|jpg|gif)$`)
+
+type StaticImageDescriptor struct {
+	Name      string
+	SizeLimit int
```

Comment:
Since you need not implement `GetSizeLimit()`, you do not need this field. Remove it.